SampleAppSwiftUI/Resources/CoinDataModel.xcdatamodeld/CoinDataModel.xcdatamodel/contents

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<model type="com.apple.IDECoreDataModeler.DataModel" documentVersion="1.0" lastSavedToolsVersion="21754" systemVersion="22F66" minimumToolsVersion="Automatic" sourceLanguage="Swift" userDefinedModelVersionIdentifier="">
+    <entity name="CoinDataCD" representedClassName="CoinDataCD" syncable="YES" codeGenerationType="class">
+        <attribute name="coinData" attributeType="Transformable" customClassName="CoinData"/>
```

Comment:
you can use "NSSecureUnarchiveFromData" in the transformable type in coredata model in order to secure transformable format. You can either type the name in the data model class, or add this line into the contents file.

`<attribute name="coinData" attributeType="Transformable" valueTransformerName="NSSecureUnarchiveFromData" customClassName="CoinData"/>`